NodeFirefox/Dockerfile

```diff
@@ -1,22 +1,24 @@
-FROM selenium/node-base:2.53.0
+FROM selenium/node-base:2.53.1
 MAINTAINER Selenium <selenium-developers@googlegroups.com>
 
 USER root
 
 #=========
 # Firefox
 #=========
-ENV FIREFOX_VERSION 45.0.2
```

Comment:
This is the version to be used rather than the version(s) to be downloaded. It made more sense to me to move it to after the downloads, but I can move it back if preferred.